Makefile

```diff
@@ -27,6 +27,13 @@ go_bin ?= $(PWD)/.work/bin
 $(go_bin):
 	@mkdir -p $@
 
+protoc_bin = $(go_bin)/protoc
+$(protoc_bin): export GOBIN = $(go_bin)
+$(protoc_bin): export PATH := $(shell go env GOPATH)/bin:$(PATH)
+$(protoc_bin): | $(go_bin)
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
+
```

Comment:
I don't think this works as intended. At least the CI fails as it doesn't find protoc

Reply:
This was wrong binary...